frontend/src/modules/Administration/views/AdministrationView.js

```diff
@@ -14,12 +14,18 @@
 import { Link as RouterLink } from 'react-router-dom';
 import { ChevronRightIcon, useSettings } from 'design';
 import { AdministrationTeams, DashboardViewer } from '../components';
+import { MaintenanceViewer } from '../../Maintenance/views/MaintenanceViewer';
+import { isModuleEnabled, ModuleNames } from '../../../utils';
```

Comment:
Change this import to relative import to absolute import ( `from utils`)

Reply:
My bad. Addressed in next update

Reply:
Fixed. Thanks! Missed these 😞